apps/wallet/src/engine/runners/solana/metadata.ts

```diff
@@ -1,25 +1,20 @@
 import { Metadata } from '@metaplex-foundation/mpl-token-metadata';
 import type { Connection } from '@solana/web3.js';
 import { PublicKey } from '@solana/web3.js';
-import { logger, Networks } from '@walless/core';
-import type { MetadataDocument } from '@walless/store';
-import { METADATA_PROGRAM_ID, solMint } from 'utils/constants';
-import { storage } from 'utils/storage';
+import type { TokenMetadata } from '@walless/core';
+import { logger } from '@walless/core';
+import { METADATA_PROGRAM_ID } from 'utils/constants';
 
 import { throttle } from './internal';
 import localMetadata from './local-metadata.json';
 
 export type GetMetadataFunc = (
```

Comment:
if this function type just meant to get token metadata, it could be changed to `GetTokenMetadataFunc`